setup.py

```diff
@@ -40,13 +40,8 @@
     packages=find_packages(exclude=['docs', 'examples', 'tests']),
     install_requires=[
         'async_generator>=1.10,<2',
-        'ipaddress>=1.0.22,<2',
         'trio>=0.11',
-        # TODO: confirm whether wsaccel is relevant to performance
-        # Disabled on pypy: https://github.com/methane/wsaccel/issues/19
-        'wsaccel>=0.6.2,<0.7;implementation_name!="pypy"',
-        'wsproto>=0.14,<0.15',
-        'yarl>=1.2.6,<2'
+        'wsproto>=0.14,<0.16',
```

Comment:
wsproto 0.15 dropped python 3.5 support.  I think it's OK, but it may affect our CI, especially with pinned dependencies coming in PR #137, where we may need different pinned dependencies for the 3.5 build.  Or finally let go of 3.5.

Reply:
Right, with pinning it will be problematic, unless pinned to a 0.14.* version.

At this point I think dropping Python 3.5 is more than reasonable...

Reply:
it is reasonable, but I'm trying to hold off until the next minor release

So far all the pending changes since trio-websocket 0.8.1 are bugfix / non-feature.  There could still be some app out there on python 3.5 / trio-websocket 0.8 that could use the bug fixes.